dapp/src/service/walletService.ts

```diff
@@ -53,4 +128,6 @@ export {
   setConnection,
   disconnect,
   initializeConnection,
+  getWalletHealth,
+  checkAndNotifyFunding,
```

Comment:
```suggestion
  export {
  loadedPublicKey,
  loadedProvider,
  setConnection,
  disconnect,
  initializeConnection,
  getWalletHealth,
};
export { checkAndNotifyFunding };
```
this will resolve the test failing issue